autobuild/tfchain.sh

```diff
@@ -26,21 +27,21 @@ TFCHAIND=$TFCHAIN/cmd/tfchaind
 TFCHAINC=$TFCHAIN/cmd/tfchainc
 BRIDGED=$TFCHAIN/cmd/bridged
 
+
 pushd $TFCHAIND
-# go build -ldflags "-linkmode external -s -w -extldflags -static" -o $FLIST/bin
-go build -o $FLIST/bin/tfchaind
+go build -ldflags "-linkmode external -s -w -extldflags -static" -o $TFCHAIN_FLIST/bin/tfchaind
 popd
 
 pushd $TFCHAINC
-# go build -ldflags "-linkmode external -s -w -extldflags -static" -o $FLIST/bin
-go build -o $FLIST/bin/tfchainc
+go build -ldflags "-linkmode external -s -w -extldflags -static" -o $TFCHAIN_FLIST/bin/tfchainc
 popd
 
 pushd $BRIDGED
-# go build -ldflags "-linkmode external -s -w -extldflags -static" -o $FLIST/bin
```

Comment:
why is the `bridged` binary included here if it is already in its own flist?

Reply:
I was expecting the tfchain flist to contain all the binaries. removed it.